lib/matplotlib/mlab.py

```diff
@@ -233,8 +229,6 @@ def stride_windows(x, n, noverlap=None, axis=0):
         The number of data points in each window.
     noverlap : int, default: 0 (no overlap)
         The overlap between adjacent windows.
-    axis : int
-        The axis along which the windows will run.
 
     References
     ----------
```

Comment:
You can probably at least remove the SO links (which predate sliding_window_view); FWIW I would just get rid of the entire function and inline it into _spectral_helper (and this way you only need to check once that noverlap < NFFT).